experimenter/tests/integration/nimbus/test_create_mobile_experiment.py

```diff
@@ -5,21 +5,25 @@
 
 import pytest
 import requests
-from nimbus.models.base_dataclass import BaseExperimentApplications
+
 from nimbus.pages.experimenter.summary import SummaryPage
 from nimbus.utils import helpers
 
 
 @pytest.fixture
-def experiment_slug():
-    return "firefox-ios-integration-test"
+def experiment_slug(application):
+    return f"firefox-{application.lower()}-integration-test-experiment"
```

Comment:
How come we hard code the app specific slugs above, but then also do it here as parameterized?

Reply:
Mainly because this "test" runs in a different scope with a different pytest `conftest.py`. So when the actual test for say ios runs in `tests/integration/nimbus/ios`, it doesn't have knowledge of the slug for use with `nimbus-cli` (we have to provide the slug to enroll). With this way its just a set slug. 

If you think we should keep the parametrized slug, I could set it as an env variable and just load that into the `nimbus-cli` call. That would give us the ability to rerun on CI.

Reply:
How come it runs in a different scope?  There isn't a way to share the same conftest?

Reply:
For android, it needs a bunch of android specific files, `grandle.py`, being one of them, and I wanted to keep them out of the normal `desktop` test configs. I guess I could put all of the conftest stuff in the test file itself so as to not have an additional `conftest.py`